src/meta/duplication/meta_duplication_service.cpp

```diff
@@ -149,15 +149,24 @@ void meta_duplication_service::add_duplication(duplication_add_rpc rpc)
                                         remote_cluster_id.get_error()));
         return;
     }
-    std::vector<std::string> clusters;
-    dsn_config_get_all_keys("pegasus.clusters", clusters);
-    if (std::find(clusters.begin(), clusters.end(), request.remote_cluster_name) ==
-        clusters.end()) {
+
+    std::string metas =
+        dsn_config_get_value_string(duplication_constants::kClustersSectionKey.c_str(),
+                                    request.remote_cluster_name.c_str(),
+                                    "",
+                                    "follower cluster meta list");
+    if (metas.empty()) {
         response.err = ERR_INVALID_PARAMETERS;
         response.__set_hint("failed to find cluster address in config [pegasus.clusters]");
         return;
     }
 
+    std::vector<rpc_address> meta_list;
+    dsn::replication::replica_helper::load_meta_servers(
+        meta_list,
+        duplication_constants::kClustersSectionKey.c_str(),
+        request.remote_cluster_name.c_str());
+
```

Comment:
```
    std::vector<rpc_address> meta_list;
    dsn::replication::replica_helper::load_meta_servers(
        meta_list,
        duplication_constants::kClustersSectionKey.c_str(),
        request.remote_cluster_name.c_str());
    if (meta_list.empty()) {
        response.err = ERR_INVALID_PARAMETERS;
        response.__set_hint("failed to find cluster address in config [pegasus.clusters]");
        return;
    }
```

Reply:
the function `load_meta_servers` will core-dump if can't find, so add judge is redundant.

I still consider whether change the `load_meta_servers` action to `no core-dump`

Reply:
Changing the bahavior of `load_meta_servers` looks better to me :)

Reply:
Ok, but this is a common function, I will put other pr

Reply:
OK. Look forward to you next pull request.